### PR TITLE
Dnn exploration

### PR DESCRIPTION
**Pull request by aug-1**

### CODE REVIEW

**Review by Pontius1007 — CHANGES_REQUESTED** [review]
Gjerne legg ved et normaliseringslag! https://keras.io/layers/normalization/

**Review by Pontius1007 — APPROVED** [review]
